modules/Bio/EnsEMBL/PipeConfig/BuildRegulationMart_conf.pm

```diff
@@ -127,14 +127,8 @@ sub pipeline_analyses {
           'base_name' => "features"
       },
       -flow_into => {
-        1 => WHEN(
-          '(#dataset# ne "dmelanogaster")' => ['AddExtraMartIndexesExternalFeatures',
-                                               'AddExtraMartIndexesPeaks',
-                                               'AddExtraMartIndexesMiRNATargetFeatures',
-                                               # 'AddExtraMartIndexesMotifFeatures',
-                                               'AddExtraMartIndexesRegulatoryFeatures'],
-          ELSE 'AddExtraMartIndexesExternalFeatures',
-        )
+        1 => ['AddExtraMartIndexesExternalFeatures',
+              'AddExtraMartIndexesRegulatoryFeatures'],
```

Comment:
Not sure about the peaks though. I think I was obliged to reintroduce the creation of this datasets for 104.

Reply:
ENSPROD-6705 states that mirna, peaks and motif features aren't needed anymore. Should I put peaks back?